lib/app/features/chat/messages/views/pages/messages_page.dart

```diff
@@ -22,6 +24,16 @@ class MessagesPage extends HookConsumerWidget {
   @override
   Widget build(BuildContext context, WidgetRef ref) {
     final isEmpty = useMemoized(() => Random().nextBool(), []);
+    final mockedDate = DateTime.now();
+    const mockedMessageAuthor = MessageAuthor(
+      name: 'Test',
+      imageUrl: 'https://picsum.photos/500/400',
+    );
+    const mockedCurrentAuthor = MessageAuthor(
+      name: 'Test',
+      imageUrl: 'https://picsum.photos/500/400',
+      isCurrentUser: true,
+    );
```

Comment:
Pass `MessageAuthor` as param or create provider if you know from where this entity comes.

Reply:
Moved to the provider.